{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/api/push.py

```diff
@@ -0,0 +1,10 @@
+from channels.layers import get_channel_layer
+
+
+async def push_message_to_user(user, json_message):
+    user_id = user.id
+    channel_layer = get_channel_layer()
+    await channel_layer.group_send(f'user-{user_id}', {
```

Comment:
can just be `{user.id}`?

Reply:
-1, we need a user namespace so that we can have non-single-user-groups without worrying about name collisions. Right?

Reply:
@davisagli I understood @cdcarter to be saying "Make it `f'user-{user.id}'`".

Reply:
Ah! Of course.